src/backend/services/logger/middleware.py

```diff
@@ -11,6 +12,7 @@
 
 class LoggingMiddleware(BaseHTTPMiddleware):
     async def dispatch(self, request: Request, call_next):
+        request_body = await request.body()
```

Comment:
this too, might cause perf issues parsing body multiple times, was using for debug